apps/events/utils.py

```diff
@@ -0,0 +1,18 @@
+def truncate_dict_items(dictionary, max_limit=100):
+    """
+    Truncate dictionary items to fit within a char limit. Returns list of (key, value)
+    tuples that fit within the limit without breaking up the key value
+    """
+    formatted_items = []
+    chars_used = 0
+    for key, val in dictionary.items():
+        key_str, val_str = str(key), str(val)
+        if chars_used + len(key_str) > max_limit:
```

Comment:
note: this won't cut off a key halfway through like it will a value. ex. if the key is `time_period` it will not do `time...`

Reply:
won't this prevent showing some keys? I think we just want to truncate the values, not the entire dict.

Reply:
ah gotcha this will prevent showing some keys-- I'll fix